scope.c

```diff
@@ -246,8 +246,12 @@ Perl_tmps_grow_p(pTHX_ SSize_t ix)
 {
     SSize_t extend_to = ix;
 #ifndef STRESS_REALLOC
-    if (ix - PL_tmps_max < 128)
-        extend_to += (PL_tmps_max < 512) ? 128 : 512;
+    SSize_t grow_size = PL_tmps_max < 512 ? 128 : PL_tmps_max / 5;
```

Comment:
Why divide 5? This seems like it grows the stack only by +20% each time - which feels like a fairly small growth ratio to me.

Reply:
It's the same as AVs and the value stack.  The save stack effectively uses 2 `((old) * 3 / 2)`.

I'm happy with some other factor since this is a small part of the cost.

Reply:
I've changed this to +50%